src/utils/scripts.ts

```diff
@@ -0,0 +1,98 @@
+import * as fs from 'fs';
```

Comment:
this file has been committed accidentally too right?

Reply:
yes